tests/nested/core20/tpm/task.yaml

```
@@ -24,7 +12,7 @@ execute: |
     execute_remote "test -e /sys/kernel/security/tpm0/binary_bios_measurements"
 
     echo "and secure boot is enabled on the nested vm"
-    retry -n 5 --wait 5 execute_remote "test-snapd-mokutil --sb-state" | MATCH "SecureBoot enabled"
+    execute_remote "xxd /sys/firmware/efi/efivars/SecureBoot-*" | MATCH '0600 0000 01'
```

can you type out the full SecureBoot-... uuid here? IIUC, there could be other SecureBoot- variables by other vendors for specific things, so this may also match other vars and not always be correct on real devices.

also can you actually add the beginning and end part of the pattern too from xxd too? I'm worried if somehow there is garbage data here that it could include this and we would see it's correct when it's actually garbage

this should be sufficient:

```suggestion
execute_remote "xxd /sys/firmware/efi/efivars/SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c" | MATCH "00000000: 0600 0000 01\s+....."
```

---

done